### ISSUE TITLE
Loinc 14170-5:  "Pituitary glycoprotein hormone.alpha subunit [Mass/volume] in Serum"

### ISSUE THREAD

**Issue by jpgourdine** [obophenotype/human-phenotype-ontology#4652]
Suggest creating a new term for Loinc: 14170-5 [Pituitary glycoprotein hormone.alpha subunit [Mass/volume] in Serum] 
New term label:Abnormal Pituitary glycoprotein hormone alpha subunit level and Decreased Pituitary glycoprotein hormone alpha subunit level 
New term comment (if any):Parent term :Abnormal Pituitary glycoprotein hormone alpha subunit level , Decreased Pituitary glycoprotein hormone PMID: 27041067
Your biocurator ID for loinc2hpo (if desired): OHSU:jpgourdine

**Comment by pnrobinson** [comment]
see https://www.ncbi.nlm.nih.gov/gene/1081

**Comment by pnrobinson** [comment]
We also need elevated, e.g., https://www.cell.com/trends/endocrinology-metabolism/fulltext/1043-2760(92)90041-X

**Comment by pnrobinson** [comment]
The increased term already exists: https://hpo.jax.org/app/browse/term/HP:0031208
Adding Abn and decreased

**Comment by pnrobinson** [comment]
See HP_0032481
